app/views/shared/dashboard.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="container">
+  <h1>Job Tracker Dashboard</h1>
+</div>
+
+<div class="container">
+  <div class="col-md-6">
+    <h3>Top <%= @top_companies.count%> Companies by Level of Interest</h3>
+
+    <table class="table table-hover">
+      <thead>
+        <tr>
+          <th>Rank</th>
+          <th>Average Level of Interest</th>
+          <th>Company</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @top_companies.each_with_index do |company, index| %>
```

Comment:
could we use an ordered list here rather than needed to manipulate a ruby variable to achieve this functionality?